### ISSUE TITLE
The loss function in lightning-uq-box/lightning_uq_box/uq_methods /prob_unet.py

### ISSUE THREAD

**Issue by qqaazz800624** [lightning-uq-box/lightning-uq-box#133]
A little user experience and a piece of suggestion to add.

The compute_loss() function defines the loss like below:

loss = -(rec_loss_sum + self.beta * kl_loss)

which I found it a little unreasonable, because the rec_loss_sum and kl_loss are two positive terms.
Based on the loss defined above, the kl_loss will become an extremely high value, such as 6e20....

<img width="242" alt="image" src="https://github.com/lightning-uq-box/lightning-uq-box/assets/79075772/047ee09c-f0e1-4704-b479-be56be05ce5a">

Perhaps, the loss function should be modified as:

loss = (rec_loss_sum + self.beta * kl_loss)

Remove the negative sign. Then everything becomes reasonable.

<img width="244" alt="image" src="https://github.com/lightning-uq-box/lightning-uq-box/assets/79075772/c7edd9e7-f999-432f-a4aa-d8aa347e9449">

**Comment by qqaazz800624** [comment]
By the way, I would like to say this library is amazing and helpful!
Thanks for the authors' hard work, maintenance and continuing updates.

**Comment by nilsleh** [comment]
Thanks a lot for the bug report @qqaazz800624 ! I have a separate PR where I was trying to test the implementation for various tasks, and observed training instabilities but didn't get around to properly debugging so thank you very much! Would you like to open a PR to fix this? 

And thank you for your kind words, we are happy to see that people start using it and grateful for any feedback (and bug fixes of course). I'd be curious what sort of problem you are applying the model to, if you don't mind me asking?

**Comment by qqaazz800624** [comment]
Yeah, I am willing to try to fix the problem and provide some feedbacks. Thanks for your response!
Actually, I can tell that you have already invested lots of effort in developing these UQ methods. And I am truly grateful that someone has already provided a framework of implementation. But I am pretty new to the Github and git commands. I will need some time to figure out how to use fork and pull request to fix it. Haha.

I was using probabilistic U-Net on medical imaging task. The data was dual energy chest X-ray. I am trying to build a probabilistic segmentation model to provide multiple possible segmentation masks. Based on that, I can try to figure out the meaning of epistemic uncertainty in segmentation task. It's pretty preliminary. I just entered this research area not long ago. It would be nice that we keep in touch to share some ideas regarding UQ research.

**Comment by nilsleh** [comment]
We have included a small page on the docs about [contributing to the repo](https://lightning-uq-box.readthedocs.io/en/latest/contribute.html), however, we could also miss details so feel free to ask questions or make suggestions how it can be more clear.

And that seems like a very interesting task, our background is Earth Observation data, however, I think there are lots of overlaps between medical imaging and EO in the way that tasks are phrased and in what way UQ can help, so I'd be happy to stay in touch over this.

**Comment by qqaazz800624** [comment]
Sure. I have just opened a pull request for you to review. 
What I did is simply removing the negative sign: 

Before: loss = -(rec_loss_sum + self.beta * kl_loss)
After: loss = (rec_loss_sum + self.beta * kl_loss)

And I think this could allow the model's optimization process work just we expected.

**Comment by nilsleh** [comment]
Closed by #135
